test/test_open.py

```diff
@@ -184,6 +202,12 @@ def test_mode_t(self, safer_open):
         assert FILENAME.read_text() == 'hello'
 
     def test_symlink_file(self, safer_open):
+        # check if we are on Windows and have admin rights
+        if os.name == 'nt' and not is_windows_admin():
```

Comment:
You should instead decorate this test case with https://docs.python.org/3/library/unittest.html#unittest.skipIf

Reply:
Thanks!
I've done this.